astropy/utils/iers/data/Leap_Second.dat

```diff
@@ -1,15 +1,15 @@
 #  Value of TAI-UTC in second valid beetween the initial value until
 #  the epoch given on the next line. The last line reads that NO
-#  leap second was introduced since the corresponding date
+#  leap second was introduced since the corresponding date 
 #  Updated through IERS Bulletin 59 issued in January 2020
-#
+#  
```

Comment:
I know the added whitespace is not ideal, but maybe best to just bundle it exactly as it is provided...